### DISCUSSION TITLE
Removing resolution information from Seurat object

### DISCUSSION THREAD

**Discussion by VaiskiVemmel** [satijalab/seurat#6831] [Q&A]
Hi.

I have a problem that while I was working with my single cell data I made several different resolutions for clustering before settling with Resolution 0.8. Later I realized that all my clustering and resolution tests were saved into the Seurat object and the original 1.1GB R-object had inflated to 9GB. 

Is there some quick way to remove these unnecessary UMAPs and metadata from the Seurat object with R? 9GB dataset is very heavy to work with.

Basically remove everything I created here:

sunion <- ScaleData(sunion, verbose = FALSE)
sunion <- RunPCA(sunion, npcs = 50, verbose = FALSE)
sunion <- RunUMAP(sunion, reduction = "pca", dims = 1:50)
sunion <- FindNeighbors(sunion, reduction = "pca", dims = 1:50)
sunion <- FindClusters(sunion, resolution = 0.6)
p1 <- DimPlot(sunion, reduction = "umap", label = TRUE, group.by = "seurat_clusters")
